### PR TITLE
Ch1284/container using volumes from must include subscription to depended upon container

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

**Comment by dexhorthy** [comment]
@laverya this looks right to me but I don't know enough about our orchestration, @emosbaugh do you wanna take a quick look?

**Review by dexhorthy — APPROVED** [review]

File: src/predicates.ts [hunk 1]
```diff
@@ -1032,3 +1032,95 @@ export class CustomRequirementsNotUnique implements Predicate<any> {
...
+          if (!_.isUndefined(event.subscriptions)) {
+            for (const subscription of event.subscriptions) {
+              subscriptionMap[subscription.component + ":" + subscription.container] = component.name + ":" + container.image_name;
+            }
+          }
+        }
+      }
+    }
+  }
+  // console.log(JSON.stringify(subscriptionMap) + "\n");
```
Comment (emosbaugh) [inline src/predicates.ts]: remove debug code...

File: src/rules/containers.ts [hunk 2]
```diff
@@ -847,4 +992,5 @@ export const all: YAMLRule[] = [
   containerClusterInstanceCountMaxUint,
   containerClusterInstanceCountDegradedUint,
   containerClusterInstanceCountHealthyUint,
+  containerVolumesSubscribtionExists,
```
Comment (emosbaugh) [inline src/rules/containers.ts]: Subscri`b`tion

**Review by emosbaugh — APPROVED** [review]

**Comment by dexhorthy** [comment]
:tada:

**Pull request merged**